src/Job/JobQueuePullerInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace AgDevelop\ForkingSupervisor\Job;
+
+interface JobQueuePullerInterface
+{
+    public function pull(): JobInterface|null;
```

Comment:
[Subiektywnie] Rzucam temat - czy przy nullu będącym przy tylko jednym typie powinniśmy stosować unie? Słyszałem opinie, że z question markiem wygląda to lepiej i jest przede wszystkim bardziej czytelne, krótsze - `?JobInterface`

Reply:
Słusznie, fixed